### ISSUE TITLE
Date, Month and Year control in calendar control has wrong control type

### ISSUE THREAD

**Issue by ylux587** [telerik/kendo-ui-core#5905]
### Bug report
Date, Month and Year control in calendar control has wrong control type

### Reproduction of the problem
Render a calendar control and then open AI4D to check control type.

### Current behavior
Date, Month and Year control in calendar control has wrong control type, which is HyperLink

![image](https://user-images.githubusercontent.com/10621694/86947772-16363200-c101-11ea-9e13-db0a0bbbc523.png)
![image](https://user-images.githubusercontent.com/10621694/86947802-20f0c700-c101-11ea-896d-d98dd35665b9.png)
![image](https://user-images.githubusercontent.com/10621694/86947836-2ea64c80-c101-11ea-84c8-3551a76b2132.png)

### Expected/desired behavior
Date, Month and Year control should have control type as button

### Environment

* **Kendo UI version:** Kendo UI version: 2019.3.1114
* **jQuery version:** 3.3.28
* **Browser:**Chromium Edge Version: 84.0.488.1 (Official build) dev (64-bit)
* **AI4D Version:** 1.1.1152.1

**Comment by dmanova** [comment]
Hello @ylux587 ,
Kendo UI tries to follow the WCAG 2.1 guidelines or give the developer the opportunity to do so with some additional configuration. As far as I could tell Accessibility Insights for Web, also does (https://accessibilityinsights.io/docs/en/web/overview).
According to [Success Criterion 2.4.4 Link Purpose](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=244#link-purpose-in-context) the purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context.

Kendo UI follows this guideline as it supports techniques, described by the WCAG 2.1 guidelines like:
- [Providing link text that describes the purpose of a link](https://www.w3.org/WAI/WCAG21/Techniques/general/G91.html)


Changing the role, hence the HTML tag to be button instead of link for the elements mentioned, although is likely to bring some clarity to accessibility, it involves major or braking changes in the current implementation of the Kendo UI widget.
